config/deploy.rb

```diff
@@ -37,6 +43,14 @@
 # set :keep_releases, 5
 
 namespace :deploy do
+  desc 'Restart application'
+  task :restart do
+    on roles(:app), in: :sequence, wait: 5 do
+      execute :touch, release_path.join('tmp/restart.txt')
```

Comment:
:ok_hand: Maybe when we rebuild exhibits, I'll get a chance to try out passenger 5 and see how it goes.